tests/providers_test.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2015 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for perfkitbenchmarker.providers"""
+
+import unittest
+
+from perfkitbenchmarker import benchmark_spec
+from perfkitbenchmarker import providers
+
+
+class LoadProvidersTestCase(unittest.TestCase):
```

Comment:
It would be nice to have another test, in addition to these, that creates a BenchmarkSpec object, calls ConstructVirtualMachines(), and verifies that it actually loads the right provider. Up to you if that's feasible, though.